jekyll/_cci2/private-images.md

```diff
@@ -20,7 +30,6 @@ jobs:
     working_directory: ~/my_app
     steps:
       - checkout
-      - setup_remote_docker
```

Comment:
I would delete this from the second example with the `machine` executor:
```
 # build and test application
      - run: |
          docker build -t my-app .
          # assuming that our app expects to have DB on localhost
          docker run --network container:db my-app test
```
and for the first example, just copy the bit from the bottom of this section: https://circleci.com/docs/2.0/configuration-reference/#docker--machine-executor